_security/access-control/authentication-tokens.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: default
+title: Authorization tokens
+parent: Access control
+nav_order: 125
+redirect_from:
+ - /security/access-control/authorization-tokens/
+ - /security-plugin/access-control/authorization-tokens/
+---
+
+Authorization tokens
+
+The Security plugin allows you to configure two types of authentication tokens: On-Behalf-Of (OBO) tokens and Service Account tokens.
+
+## On-Behalf-Of authentication
+
+The following sections describe the use, configuration, structure, and endpoint for OBO tokens.
+
+### Usage
+
+On-Behalf-Of tokens are a special form of JSON Web Token (JWT) used for managing authentication requests between a user's client and an extension. These tokens operate "just-in-time," meaning that a token is issued immediately before it is required for authentication. A token will have a configurable window of validity (with a maximum duration of five minutes), after which it expires and cannot be used.
+
+An extension can use an OBO token to interact with an OpenSearch cluster, using the same privileges as the user it represents. This is why these tokens are called "on-behalf-of." Since these tokens are not restricted, they enable services to function as though they are the original user until the token expires. This implies that the feature's applicability extends beyond just extension-related uses cases, allowing for a wider range of uses.
```

Comment:
```suggestion
An extension can use an OBO token to interact with an OpenSearch cluster, using the same privileges as the user it represents. This is why these tokens are called "on-behalf-of." Since these tokens are not restricted, they enable services to function as though they are the original user until the token expires. This implies that the feature's applicability extends beyond only extension-related uses cases, allowing for a wider range of uses.
```